### ISSUE TITLE
Does not work with current ember beta

### ISSUE THREAD

**Issue by topaxi** [freshbooks/ember-responsive#37]
Tried using ember-responsive with ember `2.1.0-beta.2` and it throws the following in [`responsive.js:3`](https://github.com/freshbooks/ember-responsive/blob/master/addon/instance-initializers/responsive.js#L3):

```
Uncaught TypeError: instance.container.lookupFactory is not a function
```

**Comment by marcemira** [comment]
:+1:  Same here

**Comment by jasonmit** [comment]
I opened a PR with a fix

**Comment by f3ndot** [comment]
@topaxi @marcemira Fixed in version 1.1.1
